apps/shell/Kconfig

```diff
@@ -26,7 +26,7 @@ config TASH_USLEEP
 config TASH_REBOOT
 	bool "Enable 'reboot' command in TASH"
 	default n
-	depends on BOARDCTL_RESET
+	depends on BOARDCTL_RESET || ARCH_ARMV7A
```

Comment:
This looks odd. How are the other arch and other boards working with TASH_REBOOT. Please implement accordingly for V7A also.